src/components/board/Board.jsx

```diff
@@ -1,4 +1,17 @@
+import Image from "next/image";
+import BoardIMG from "@/public/board/Exec_Board.webp";
+
 const Board = () => {
-  return <div></div>;
+  return (
+    <div className="bg-witl-purple-300 flex flex-col pb-40">
+      <Image src={BoardIMG} alt="Board" className="self-end mr-10 mt-10" />
+      <p
+        className="text-witl-purple-100 w-min text-8xl absolute z-0"
+        style={{ bottom: "4%", left: "9%" }}
```

Comment:
let's use the tailwind classes with top-[xx%], etc. instead of using the reg CSS styling.